tests/test_base_models.py

```diff
@@ -511,6 +519,7 @@ def _test_featurize_sequence(self, model_fn):
     prev_features = [None] * len(test_input)
     for max_len in [32, 64, 128, 256]:
         chunk_config = self.default_config()
+        chunk_config["chunk_long_sequences"] = True
```

Comment:
This change is needed because now the default for TextCNN (and FastTextCNN) is to have `chunk_long_sequences` set to False